xonsh/xonfig.py

```diff
@@ -662,31 +723,48 @@ def _xonfig_create_parser():
         "style", nargs="?", default=None, help="style to preview, default: <current>"
     )
     subp.add_parser("tutorial", help="Launch tutorial in browser.")
+    kern = subp.add_parser("jupyter_kernel", help="Generate xonsh kernel for jupyter.")
```

Comment:
Please call this `jupyter-kernel`. We generally use dashes for CLIs

Reply:
... Now he tells me...